acc_test/resource_kafka_user_test.go

```diff
@@ -30,19 +32,25 @@ func TestKafkaUserResource(t *testing.T) {
 	kafkaUsername3 := "charlie3"
 	oldPassword := "charlie123!"
 	newPassword := "charlie123standard!"
-	zookeeperNodeSize := "zk-developer-t3.small-20"
+	kafkaNodeSize := "KFK-DEV-t4g.medium-80"
+	zookeeperNodeSize := "KDZ-DEV-t4g.small-30"
+	kafkaVersion := "apache-kafka:2.7.1.ic1"
```

Comment:
can we use artefact version? (`2.7.1`) because if we update this, and deprecate `ic1` (which is technically our internal versioning system) it will break the test

Reply:
the problem is the asymmetry with the return value thing, if I give 2.7.1 the plan will not be empty because the API returns apache-kafka:2.7.1.ic1